Dockerfile

```diff
@@ -15,16 +15,16 @@ FROM prebuild as build
 RUN /bin/bash -c "mkdir -p $(go env GOPATH)/src/github.com/cockroachdb && \
     cd $(go env GOPATH)/src/github.com/cockroachdb"
 WORKDIR /go/src/github.com/cockroachdb
-RUN /bin/bash -c "git clone --branch v21.2.4 https://github.com/cockroachdb/cockroach"
+RUN /bin/bash -c "git clone --branch v22.1.1 https://github.com/cockroachdb/cockroach"
 WORKDIR /go/src/github.com/cockroachdb/cockroach
-RUN /bin/bash -c "git submodule update --init --recursive && make build && \
-    	      make install"
+RUN /bin/bash -c "git submodule update --init --recursive"
+RUN /bin/bash -c "NODE_OPTIONS=--max-old-space-size=4096 make build && make install"
```

Comment:
It was failing with an out of memory error until I set `max-old-space-size` to give node 4GB memory for compilation.